code-styling/markdown/README.md

```diff
@@ -0,0 +1,1155 @@
+
+## 4. Blockquotes
+
+- Blockquotes MUST use the greater-than (`>`) character.
+
+- The greater-than character MUST be followed by a single space.
+
+  _Correct example_:
+  
+  ```Markdown
+  > This is a pretty nice blockquote :)
+  ```
+  _Incorrect example_:
+  
+  ```Markdown
+  >This blockquote is lacking some space.
+  ```
+
+- Consecutive lines within a blockquote MUST also start with the greather-than (`>`) character.
+
+  _Correct example_:
+  
+  ```Markdown
+  > I told my wife she was drawing her eyebrows too high.
+  > She looked surprised.
+  ```
+  _Incorrect example_:
+  
+  ```Markdown
+  > I tried to catch fog yesterday.
+  Mist.
+  ```
+
+- Empty lines within a blockquote MUST use the greater-than (`>`) characater.
+
+  _Correct example_:
+  
+  ```Markdown
+  > Parallel lines have so much in common.
+  >
+  > It's a shame they'll never meet.
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  > You're not completely useless...
+  
+  > You can always serve as a bad example.
+  ```
+
+## 5. Lists
+
+- Lists MUST be preceded and followed by a newline.
+
+  _Correct example_:
+  
+  ```Markdown
+  Shopping list:
+  
+  - Milk
+  - Bread
+  - Cheese
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  Shopping list:
+  - Milk
+  - Bread
+  - Cheese
+  ```
+
+- The first level of items in a list MUST NOT be indented.
+
+  _Correct example_:
+  
+  ```Markdown
+  Shopping list:
+  
+  - Milk
+  - Bread
+  - Cheese
+  ```
+
+  _Incorrect example_:
+  
+  ```Markdown
+  Shopping list:
+  
+    - Milk
+    - Bread
+    - Cheese
+  ```
+
+- Subitems of a list MUST be indented by 2 spaces and MUST be preceded and followed by a newline.
+
+  _Correct example_:
+  
+  ```Markdown
+  - First level
+  
+    - First subitem
+    - Second subitem
+    
+  - Second item
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  - First level
+    - First subitem
+    - Second subitem
+  - Second level
+  ```
+
+- When one or more items in a list spans more than one line, the list is REQUIRED to have newlines between each item. 
+
+  _Correct example_:
+  
+  ```Markdown
+  - First item
+  
+  - This second item spans two lines
+    Not because this is needed, but hey, we need an example.
+    
+  - Third item
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  - First item
+  - This second item spans two lines
+    Not because this is needed, but hey, we need an example.
+  - Third item
+  ```
+
+- The indentation what comes inside list MUST be same as the item it is in.
+
+  _Correct example_:
+  
+  ```Markdown
+  - This item spans two lines
+    This is the second line
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  - This item spans two lines
+   This is the second line
+  ```
+
+- Each list item MUST have the same case as it would have if
+  it were concatenated with the sentence that comes before the list.
+
+  _Correct examples_:
+  
+  ```Markdown
+  I want to eat:
+
```

Comment:
In this case I tough the incorrect examples would be obvious. It would be the reverse of the correct example.

How about changing the title from 'Correct examples' to 'Examples'? (this is done on multiple occasions in this document).